Lambda-scenarios/datasync_testing.py.txt

```diff
@@ -0,0 +1,164 @@
+
+#
+# Lambda function used to write inbound IoT sensor data to RDS PostgeSQL database
+#
+import sys
+import os
+import json
+import psycopg2
+import boto3
+import base64
+from botocore.exceptions import ClientError
+import logging
+from datetime import datetime
+
+secret_name = "arn:aws:secretsmanager:eu-west-2:063935053328:secret:bip_db_pass-b2Yqp4"
+# region_name = os.environ['AWS_REGION']
+
+# Enable the logger
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.DEBUG)
+ch = logging.StreamHandler()
+ch.setLevel(logging.INFO)
+formatter = logging.Formatter("%(asctime)s - %(levelname)s - %(message)s", datefmt='%Y-%m-%d %H:%M:%S %Z')
+ch.setFormatter(formatter)
+logger.addHandler(ch)
+
+sampleAPIJSON={"sourceLocation": "/ghdevhome/binfs/210303_A01021_0222_BHY2V2DSXY/",

```

Comment:
Expect the parameters from even, no need to hard code the json..
source = event['queryStringParameters']['src']
dest = event['queryStringParameters']['dest']